InformationScripting/src/query_prompt/QueryBuilder.cpp

```diff
@@ -89,7 +95,14 @@ Query* QueryBuilder::visitOperator(QueryBuilder* self, OperatorQueryNode* op)
 	else if (leftComposite)
 	{
 		// union
-		connectQueriesWith(composite, leftComposite, new UnionOperator(), right);
+		Query* unionQuery = nullptr;
+		if (op->op() == OperatorQueryNode::OperatorTypes::Pipe)
+			unionQuery = composite->addQuery(std::unique_ptr<Query>(new UnionOperator()));
```

Comment:
Well, it also performs a split. We should either make the name more precise/complete or at least make it consistent with the rest of the names, which use `pipe`. Maybe we can come up with an even better name that achieves both purposes, something like `MagicPipeOperator`, but less "magical" :D Any ideas, or do you insist on the current name?